### PR TITLE
Fragment CSS and Refactored CSS Styling

### PR DESCRIPTION
**Pull request by keitaroryu**
- Fragmented navbar/header. Replaced navbar/header with fragmented code on the following:
>- owner-login
>- owner-registration
>- owner-home
>- results
- Refactored results.html for improved CSS styling.  Also got rid of the dead space before the map, and made the table height auto adjust.
- Got rid of the dead space before the map, and made the table height auto adjust on user-view.
- Add new default search text to instruct the user what they can do at index.html.
- Got rid of table header on results.html for redudancy.
- Fixed pages with default page headers (index shows "Test-Page" on tab).

### CODE REVIEW

File: Happy-Hour/src/main/resources/templates/owner-registration.html [hunk 1]
```diff
@@ -4,44 +4,40 @@
...
-                            <input class="form-control" th:field="${registerFormDTO.password}" type="password"/>
-                        </label>
-                        <p class="error" th:errors="${registerFormDTO.password}"></p>
-                    </div>
-                    <div class="form-group">
-                        <label>Verify Password
-                            <input class="form-control" th:field="${registerFormDTO.verifyPassword}" type="password"/>
-                        </label>
-                    </div>
+        <div class="container h-100">
```
Comment (jess-ward) [inline Happy-Hour/src/main/resources/templates/owner-registration.html]: Registration form is not centered anymore. We can get rid of line 13

File: Happy-Hour/src/main/resources/templates/owner-registration.html [hunk 2]
```diff
@@ -4,44 +4,40 @@
...
+            <div class="d-flex justify-content-center">
```
Comment (jess-ward) [inline Happy-Hour/src/main/resources/templates/owner-registration.html]: rename line 14 div class="row d-flex justify-content-center"

File: Happy-Hour/src/main/resources/templates/owner-registration.html [hunk 3]
```diff
@@ -4,44 +4,40 @@
...
+                <div class="row h-100 justify-content-center align-items-center">
```
Comment (jess-ward) [inline Happy-Hour/src/main/resources/templates/owner-registration.html]: and remove line 15. Making these changes should recenter the registration form and place it below the header and navbar

File: Happy-Hour/src/main/resources/templates/owner-login.html [hunk 4]
```diff
@@ -4,36 +4,32 @@
...
-            <div class="form-group">
-                <label>Password
-                    <input class="form-control" th:field="${loginFormDTO.password}" type="password">
-                </label>
-                <p class="error" th:errors="${loginFormDTO.password}"></p>
-            </div>
+        <div class="row d-flex justify-content-center">
+            <form method="post">
+                <div class="form-group">
+                    <label>Username</label>
```
Comment (jess-ward) [inline Happy-Hour/src/main/resources/templates/owner-login.html]: extra closing label after Username

**Review by jess-ward — CHANGES_REQUESTED** [review]
I noticed the registration page form was no longer centered and below the header and navbar fragments you added. I think the changes I have suggested here can fix that.

File: Happy-Hour/src/main/resources/templates/owner-registration.html (hunk 1 above)
Comment (keitaroryu) [reply Happy-Hour/src/main/resources/templates/owner-registration.html]: Deleted div element.

File: Happy-Hour/src/main/resources/templates/owner-registration.html (hunk 2 above)
Comment (keitaroryu) [reply Happy-Hour/src/main/resources/templates/owner-registration.html]: adjusted the class boot strap styling

File: Happy-Hour/src/main/resources/templates/owner-registration.html (hunk 3 above)
Comment (keitaroryu) [reply Happy-Hour/src/main/resources/templates/owner-registration.html]: removed div element.

File: Happy-Hour/src/main/resources/templates/owner-login.html (hunk 4 above)
Comment (keitaroryu) [reply Happy-Hour/src/main/resources/templates/owner-login.html]: removed extra ending label tag

**Comment by keitaroryu** [comment]
Thanks for the review Jessica. It was a nice catch as I didn't bother checking registration page after making my test username. Everything should be fixed and cleaned up.  Your suggestions worked like a charm.

**Pull request merged**